File: Sliding-Window/2027. Minimum Moves to Convert String/minimumMovesToConvertString.py
class Solution:
    def minimumMoves(self, s: str) -> int:
        output = 0
        i = 0
        
        while i < len(s):
            if s[i] == "X":
                output += 1
                i += 3
            else:
                i += 1
        
        return output

File: Sliding-Window/2027. Minimum Moves to Convert String/minimumMovesToConvertString.js
/**
 * @param {string} s
 * @return {number}
 */
 var minimumMoves = function(s) {
    let output = 0;
    let index = 0;
    
    while (index < s.length) {
        if (s[index] === "X") {
            output += 1;
            index += 3;
        } else {
            index += 1;
        }
    }
    
    return output;
};